lib/gateways/fixed_data_gateway.rb

```diff
@@ -0,0 +1,5 @@
+class FixedDataGateway
+  def save(fixed_data)
+    FixedDatum.create!(fixed_data)
```

Comment:
Given we don't intend to use the value returned by this, I would explicitly return `nil` to avoid leaking ActiveRecord objects.